app/controllers/peer_reviews_controller.rb

```diff
@@ -83,9 +85,10 @@ def assign(reviewer_groups, reviewee_groups)
   def unassign(reviewers_to_remove_from_reviewees_map)
     reviewers_to_remove_from_reviewees_map.each do |reviewee_id, reviewer_id_to_bool|
       reviewer_id_to_bool.each do |reviewer_id, dummy_value|
+        # find the PR that this reviewer made on this reviewee's submission
         reviewee_group = Grouping.find_by_id(reviewee_id)
-        result_id = reviewee_group.current_submission_used.get_latest_result.id
-        pr = PeerReview.find_by(result_id: result_id, reviewer_id: reviewer_id)
+        reviewee_prs = reviewee_group.peer_reviews
```

Comment:
You can add a `.find(reviewer_id: reviewer_id)` to this line instead of making it a separate check below